strongbox-security/strongbox-authentication-providers/strongbox-default-authentication-provider/src/main/java/org/carlspring/strongbox/authentication/api/impl/defaulting/DefaultAuthenticator.java

```diff
@@ -1,4 +1,4 @@
-package org.carlspring.strongbox.authentication.api.impl;
+package org.carlspring.strongbox.authentication.api.impl.defaulting;
```

Comment:
Yeah, I thought so too, but unfortunately `default` is a [Java Language Keyword ... ](https://docs.oracle.com/javase/tutorial/java/nutsandbolts/_keywords.html)